Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -423,8 +423,33 @@ struct DQEventMixing {
           }
         } // end for (muon2)
       }   // end for (muon1)
-    }     // end for (event combinations)
-  }       // end process()
+
+      for (auto& track1 : tracks1) {
+        for (auto& muon2 : muons2) {
+          twoTrackFilter = track1.isBarrelSelected() & muon2.isMuonSelected();
+
+          if (!twoTrackFilter) { // the tracks must have at least one filter bit in common to continue
+            continue;
+          }
+          VarManager::FillPair(track1, muon2, fValues, VarManager::kElectronMuon);
+
+          for (int i = 0; i < fCutNames.size(); ++i) {
+            if (twoTrackFilter & (uint8_t(1) << i)) {
+              if (track1.sign() * muon2.sign() < 0) {
+                fHistMan->FillHistClass(Form("PairsEleMuMEPM", fCutNames[i].Data()), fValues);
```

Comment:
here you need to add the electron cut name into the name of the electron-muon pair hist class